stdlib/public/SDK/os/os_log.m

```diff
@@ -71,7 +71,7 @@
 
 typedef struct {
     os_log_fmt_cmd_flags_t cmd_flags : 4;
-    os_log_fmt_cmd_type_t  cmd_type : 4;
+    os_log_fmt_cmd_type_t cmd_type : 4;
```

Comment:
These files are already not formatted Swift-ly; I don't think you should bother editing them for formatting.